assured-workloads-example/README.md

```diff
@@ -0,0 +1,122 @@
+# Assured Workloads
+
+## Overview
+
+Example of Assured Workloads using Regional Control, HSM Key and Access Justification.
+
+## Prerequisites
+
+These sections describe requirements to run this example.
+
+### Software
+
+Install the following softwares:
+
+- [Google Cloud SDK](https://cloud.google.com/sdk/install)
+- [Terraform](https://www.terraform.io/downloads.html)
```

Comment:
is there any version constraint?

Reply:
Works with Google Cloud SDK 484.0.0 and Terraform v1.9.1.